frontend/src/model/ModelReleaseDisplay.tsx

```diff
@@ -0,0 +1,135 @@
+import { Close, Done, HourglassEmpty } from '@mui/icons-material'
+import { Box, Divider, Stack, Tooltip, Typography } from '@mui/material'
+import { green, orange, red } from '@mui/material/colors'
+import { useTheme } from '@mui/material/styles'
+import { ApprovalStates } from 'types/types'
+
+import Link from '../Link'
+
+export default function ModelReleaseDisplay({
+  modelUuid,
+  release,
+  latestRelease,
+}: {
+  modelUuid: string
+  release: any
+  latestRelease: string
+}) {
+  const theme = useTheme()
+
+  function formatDate(timestamp) {
+    const date = new Date(timestamp)
+    const year = date.getFullYear().toString()
+    const formattedYear = `'${year.substring(date.getFullYear().toString().length - 2)}`
+    return `${date.getDate()} ${date.toLocaleString('default', { month: 'short' })} ${formattedYear}`
+  }
+
+  function approvalStatus(status: ApprovalStates, label: string) {
+    switch (status) {
+      case ApprovalStates.Accepted:
+        return (
+          <Tooltip title={`${label} has approved this release`}>
+            <Done htmlColor={green[500]} />
```

Comment:
Could these three icons use theme colours? Success, warning and error are green, orange and red.